Lizarov, Anna.Assignment#3.Rmd

```diff
@@ -0,0 +1,216 @@
+---
+title: 'Lizarov, Anna. Assignment #3'
+author: "Anna Lizarov"
+date: "October 30, 2018"
+output: html_document
+---
+
+## Part I
+```{r}
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+```
+
+```{r}
+(K1 <- read.csv("Class_Motivation.csv", header=TRUE))
+```
+
+```{r}
+(K2 <- K1[,-1])
+```
+
+```{r}
+K3 <- na.omit(K2) #This command create a data frame with only those people with no missing values. It "omits" all rows with missing values, also known as a "listwise deletion". EG - It runs down the list deleting rows as it goes.
+
+```
+
+```{r}
+#Standardizing values
+K3 <- scale(K3)
```

Comment:
You may wish to add a note about why you performed the standardization so in the future if you need to run this method again you can be reminded.